backend/web.config

```diff
@@ -35,7 +35,11 @@
         </rule>
       </rules>
     </rewrite>
-    
+
+    <staticContent>
+        <mimeMap fileExtension="." mimeType="text/plain" />
```

Comment:
This was just a thing I needed to host files without an extension so that we could prove identity to get an SSL certificate.

Reply:
No longer needed, but good for renewals.